site/docs/about/roadmap.mdx

```diff
@@ -9,79 +9,28 @@ import LargeParagraph from "../../src/components/LargeParagraph";
 # Roadmap
 
 <LargeParagraph>
-   This is the roadmap of Helsinki Design System. The roadmap is subject to change according to feedback and feature requests we receive.
+   This is the roadmap of Helsinki Design System. The roadmap is subject to change according to feedback and feature requests we receive. More details are added when the time approaches.
 </LargeParagraph>
 
-**Q2 2020**
-
-- ~~**Release of the first design kit**~~
-- ~~Component: Dropdown~~
-- ~~Component: Radio button~~
-- ~~Component: Checkbox~~
-- ~~Component: Text area~~
-- ~~Component: Status label~~
-- ~~Tokens: Spacing~~
-- ~~Tokens: Line heights~~
-- ~~Tokens: Breakpoints and maximum content width~~
-
 
 - **HDS Alpha release 26.05.2020**
 
-
-**Q3 2020**
-
-- ~~Update: Cleaning typography styles~~
-
-
-- ~~Component: Navigation~~
-- ~~Component: Notification~~
-- ~~Component: Logo~~
-- ~~Component: Footer~~
-- ~~Visual assets: Empty content placeholders~~
-
-
-**Q4 2020**
-
-- ~~Update: Cleaning color styles~~
-- ~~Update: Multiselect dropdown update (Combobox and Select)~~
-
-
-- ~~Component: Tooltip~~
-- ~~Component: Card~~
-- ~~Pattern: Forms~~
-- ~~Visual assets: New and updated icons~~
-
-
-- ~~**Accessibility: Audit for all released components**~~
-
-
 - **HDS Beta release 29.10.2020**
 
-
-- ~~Component: Tag~~
-- ~~Component: Loading spinner~~
-- ~~Component: Selection group~~
-- ~~Component: Page container~~
-- ~~Component: Search field~~
-- ~~Pattern: Form validation~~
-- ~~Component: Accordion~~
-- ~~Component: Tabs~~
-
+For previously released features, see changelogs in [What is new](/about/new)
```

Comment:
For accessibility, HDS doc site often describes links more elaborately.

For example; "For previously released features, see changelogs in the [What is new section](/about/new)."